pipelines/pipelines/cre-gitrepo.yaml

```diff
@@ -57,51 +56,25 @@ spec:
         - name: url
           value: $(params.url)
 
-    - name: extract-jupyterhub-namespace
-      taskRef:
-        name: extract-external-service
-      params:
-        - name: externalServices
-          value: "$(params.externalServices)"
-        - name: serviceName
-          value: jupyterhub
-        - name: serviceProperty
-          value: namespace
-
     - name: buildah
       taskRef:
         name: buildah
         kind: ClusterTask
       runAfter:
         - generate
-        - extract-jupyterhub-namespace
       workspaces:
-        - name: data
+        - name: source
           workspace: data
-        - name: sslcertdir
-          workspace: sslcertdir
       params:
         - name: IMAGE
-          value: image-registry.openshift-image-registry.svc:5000/$(tasks.extract-jupyterhub-namespace.results.value)/$(context.pipelineRun.name)
-
-    - name: resolve-owner-reference
-      taskRef:
-        name: resolve-owner-reference
-      runAfter:
-        - extract-jupyterhub-namespace
-      params:
-        - name: ownerReferences
-          value: $(params.ownerReferences)
-        - name: namespace
-          value: $(tasks.extract-jupyterhub-namespace.results.value)
+          value: image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/$(context.pipelineRun.name)
 
     - name: create-image-stream
       taskRef:
         name: openshift-client
         kind: ClusterTask
       runAfter:
         - buildah
```

Comment:
We don't need to run after buildah do we ? ImageStream creation and image can be parallel (since we're removing runAfter members...)

Reply:
Well... the logic in the original pipeline, I believe, was that we wouldn't create the ImageStream unless the build completes.

Here this is inconsistent with how the other pipelines work. But running it in parallel has a problem, given how the imagestream(tag) is currently used/expected to be: the pipeline would have to have a final step to update the "phase" annotation.

I'll give it a closer look

Reply:
`                opendatahub.io/notebook-image-phase: Succeeded`
This one ?
Hum. That does not really belong here, does it ? that's a status item, should not be in metadata, and probably not in ImageStream at all, I think.

Reply:
> that's a status item, should not be in metadata, and probably not in ImageStream at all, I think.

Agreed, we should move that to the CRE - and as contitions and not phase.

But for now this is currently being used/expected by the dashboard, see e.g. https://github.com/thoth-station/odh-dashboard/pull/14#issuecomment-1384423533. And it's not just the gitrepo-related imagestreams, import also works the same for now